droidlet/memory/craftassist/mc_memory.py

```diff
@@ -197,6 +202,12 @@ def upsert_block(
             cmd, memid, b, m, self.get_time(), player_placed, agent_placed, ref_type, x, y, z
         )
 
+    def check_inside(self, mems):
```

Comment:
hm how this even work if both methods have the same name ?

Reply:
its a different namespace? the check_inside called in the method is from perception... to have "the same name" I think it would need to be self.check_inside()